eng/scripts/Invoke-DependencyCheck.ps1

```diff
@@ -0,0 +1,69 @@
+Param(
+    [string] $newPackageDirectory
+)
+
+. (Join-Path $PSScriptRoot .. common scripts common.ps1)
+
+$ignoreCheck = ,"sdk/synapse/azartifacts"
+
+$sdks = Get-AllPackageInfoFromRepo
+
+## Create depcheck module
+$workingPath = Join-Path $RepoRoot "sdk" "depcheck"
+if (Test-Path -Path $workingPath)
+{
+    Remove-Item -Path $workingPath -Recurse -Force
+}
+New-Item -ItemType Directory -Force -Path $workingPath
+
+## Init go mod
+Set-Location $workingPath
+Write-Host "##[command]Executing go mod init in " $workingPath
+go mod init github.com/Azure/azure-sdk-for-go/sdk/depcheck
+if ($LASTEXITCODE) { exit $LASTEXITCODE }
+
+# Get all existed packages
+$packagesImport = ""
+foreach ($sdk in $sdks)
+{
+    if ($sdk.Name -like "*internal*" -or $sdk.Name -in $ignoreCheck)
+    {
+        continue
+    }
+    if ((Resolve-Path $sdk.DirectoryPath).Path -eq (Resolve-Path (Join-Path $RepoRoot $newPackageDirectory)).Path)
```

Comment:
The slashes shouldn't be an issue when comparing the $sdk.name $newPackageDirectory as they should match exactly or if you are worried you can just normalize them by replacing them.

Reply:
Fixed.